### DISCUSSION TITLE
Multiple worlds and panning

### DISCUSSION THREAD

**Discussion by jatech1** [maplibre/maplibre-gl-js#3702] [Q&A]
By default, MapLibre allows multiple copies of the world to fill map left to right.  If this is turned off with setRenderWorldCopies, it shows only one world, but then allows zoom out to the point that the latitude bound hits the limit, causing blank map borders on the left and right.  Google maps, and most other implementations have one world but only allow zoom out until the longitude bound hits the limit, and then requires panning vertically to see the rest of the map... thereby always maintaining a filled map container and never showing blank areas on sides.  And it still allows panning horizontally around the one world copy without limit.  Am I missing something, or does MapLibre have no way to achieve this very simple and common map mode?

**Comment by HarelM** [comment]
I think you can also set bounds and min/max zoom to achieve this, can't you?
I must admit I haven't tried achieving it, but I believe it should be possible...?

**Comment by jatech1** [comment]
Don't want to limit zoom... still want to be able to go from 0 to 22... but just want the map to always fill the container and no world wrap.  Have not found any combination that works for that.
